dbus-codegen/src/main.rs

```diff
@@ -34,6 +34,9 @@ fn main() {
              .help("If present, connects to the supplied service to get introspection data. Reads from stdin otherwise."))  
         .arg(clap::Arg::with_name("path").short("p").long("path").takes_value(true).value_name("PATH")
              .help("The path to ask for introspection data. Defaults to '/'. (Ignored if destination is not specified.)"))
+        .arg(clap::Arg::with_name("filter").short("f").long("filter").takes_value(true).value_name("FILTER")
+            .help("Comma separated list of filter strings. Only matching traits are generated. \
```

Comment:
should probably be named `--interfaces` instead of `--filter`, this would be more clear, especially in case I want to add more filters in the future.